ee/tables/jwt/table.go

```diff
@@ -0,0 +1,177 @@
+package kolide_jwt
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// I've set 3 different states for if the signature is verified:
+// VALID - The token parsed without errors and the signature was successfully validated.
+// INVALID - The signature attempted to validate with the matched public key, but it was a bad key.
+// UNKNOWN - The default state. This can mean that no key id matched, or simply no keys were provided to validate against.
+const (
+	Valid   = "client_valid"
+	Invalid = "client_invalid"
+	Unknown = "client_unverified"
+)
+
+// Values for include_raw_jwt column.
+var (
+	allowedIncludeValues = []string{"true", "false", "1", "0"}
```

Comment:
Honestly, reading the code we probably just need `trueIncludeValues = []string{"true", "1"}` right?

Reply:
Yeah - updated.